build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
   }
   dependencies {
     classpath "com.google.protobuf:protobuf-gradle-plugin:0.7.3"
-    classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.2"
+    classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.3.2"
```

Comment:
I don't think this actually changes the version of the formatter we use to format our code.

We use formatter in two places. One to format and verify Java code in toolkit itself and another to format the java files we render. If we look at the [build log](https://travis-ci.org/googleapis/toolkit/jobs/164732499#L661), we can see that the format verification is still done with version 1.0. I have no idea why the formatting in the comment changed. It's possible that the gradle plugin adds a new option to the formatter.

To change the version of the formatter we use to format toolkit, I think we need to change [this line](https://github.com/garrettjonesgoogle/toolkit/blob/7a75b5aa164d18026de8acc08ebf4366b0598ff5/build.gradle#L446)